src/java.desktop/macosx/classes/com/apple/laf/AquaScrollBarUI.java

```diff
@@ -527,6 +527,21 @@ void setScrollByBlock(final boolean block) {
         }
 
         public void actionPerformed(final ActionEvent e) {
+            Component parent = fScrollBar.getParent();
+            do {
+                if (parent instanceof javax.swing.JFrame par) {
+                    if (!par.isEnabled()) {
+                        ((Timer)e.getSource()).stop();
```

Comment:
```suggestion
                        ((Timer) e.getSource()).stop();
```

Reply:
same spacing issue in other places too as mentioned above...

Reply:
Ok... as you are changing the file I guess you can expand the wild imports, It's up to you.

Reply:
ok